### ISSUE TITLE
Pitch bend range

### ISSUE THREAD

**Issue by ryukau** [magnetophon/DigiDrie#31]
Just to make sure, MIDI supports pitch bend range (pitch bend sensitivity in specification). We don't use it? I'm OK to ignore MIDI pitch bend range. Current implementation is more flexible.

Also is it possible to make `positive bend range` and `negative bend range` as float?

**Comment by magnetophon** [comment]
I didn't know that.
I'm OK with using it, if you want.

We can make the ranges float, but why would you want that?
Don't you usually want to bend to a certain note?

**Comment by ryukau** [comment]
Thanks for the answer. I'll go with current implementation. Different bend range for up/down is nice to have.

The reason I asked for float is that my C++ code doesn't have a mechanism to pass negative int between UI and DSP. The easiest solution is make it float, and apply `floor` or equivalent. Though I should just implement negative integer properly. I'll work on it.

I made everything as unsigned because overflow of int is undefined in C++. I was surprised when I learned that.

- [c++11 - Is signed integer overflow still undefined behavior in C++? - Stack Overflow](https://stackoverflow.com/questions/16188263/is-signed-integer-overflow-still-undefined-behavior-in-c)
